services/webhook/deliver.go

```diff
@@ -113,16 +113,21 @@ func Deliver(t *models.HookTask) error {
 		signatureSHA256 = hex.EncodeToString(sig256.Sum(nil))
 	}
 
+	event := t.EventType.Event()
+	eventSingle := string(t.EventType)
 	req.Header.Add("X-Gitea-Delivery", t.UUID)
-	req.Header.Add("X-Gitea-Event", t.EventType.Event())
+	req.Header.Add("X-Gitea-Event", event)
+	req.Header.Add("X-Gitea-Event-Single", eventSingle)
```

Comment:
What about 
```suggestion
	req.Header.Add("X-Gitea-Event-Type", eventSingle)
```
with corresponding variable name change instead?
Are the other providers expecting a "--Single" header?
If that is the case then I would only rename the header for Gitea and for none else.

Reply:
There is no renaming being done, this is a new header. I only added them for the other forges "just in case:tm:"

I do like `-Type` more, though.